docs/ops/python_shell.md

```diff
@@ -28,7 +28,7 @@ It can be used in a local setup as well as in a cluster setup.
 To use the shell with an integrated Flink cluster just execute:
 
 {% highlight bash %}
-bin/pyflink-shell.sh local
+pyflink-shell.sh local
```

Comment:
@jingwen-ywb Thanks a lot for the PR. Good job!

I think the reason that to remove the `bin` is we can install pyflink with Pypi so that the pyflink-shell.sh is no the system PATH automatically. In this case, it's better to also add the document about how to install pyflink with Pypi.. Meanwhile, we can remove the document about installing apache-beam since it will be installed during install pyflink. 

What do you think?

Reply:
yes，I can't agree with you more. Pyflink provides a very convenient way to development - pyflink shell After successfully executing python -m pip install apache-flink。